src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function makeAdder(num) {
+    if (num || num === 0) {
+      sum += num;
+
+      return makeAdder;
+    }
+
+    const RESULT = sum;
```

Comment:
```suggestion
    const result = sum;
```
it's not the constant